ha/setup/ha_setup.py

```diff
@@ -702,9 +708,10 @@ def _destroy_cluster(self, node_name: str):
         Args:
             node_name (str): Node name
         """
-        Log.error(f"Destroying the cluster on {node_name}.")
+        Log.info(f"Destroying the cluster on {node_name}.")
+        output = self._execute.run_cmd(const.PCS_CLUSTER_KILL)
```

Comment:
Not sure if this will help as process will be restarted but we can try and see the result.

Reply:
PCS_CLUSTER_KILL
it will kill cluster so it will not be restarted
to recover we need to do pcs cluster start to start service

Reply:
It will kill pacemaker/corosync services which are managed by systemd. systemd will restart those.